3.9-minimal/s2i/bin/run

```diff
@@ -17,7 +17,9 @@ function should_migrate() {
 
 # Guess the number of workers according to the number of cores
 function get_default_web_concurrency() {
-  limit_vars=$(cgroup-limits)
+  # Using python and just python here because the script has
+  # platform-python in its shebang which we don't have in the minimal image
+  limit_vars=$(python /usr/bin/cgroup-limits)
```

Comment:
python3.9 is the same as python here because the main virtual environment is already activated. This way, it's more universal and less error-prone but it will be generated in the future hopefully so the second argument does not matter that much. Should I change it?

Reply:
[Facepalm] Of course, I forgot we're in the virtual environment here. Makes sense, don't change anything.